packages/twenty-e2e-testing/.env.example

```diff
@@ -1,5 +1,5 @@
 # Note that provide always without trailing forward slash to have expected behaviour
-FRONTEND_BASE_URL=http://localhost:3001
+FRONTEND_BASE_URL=http://app.localhost:3001
```

Comment:
Why did we change the URL to this? Maybe I missed something.

Reply:
@Devessier it's because of multiworkspace feature now, if I type `localhost:3001` it's redirected to `app.localhost:3001` so I changed it